Src/EB/AMReX_EB_STL_utils.H

```diff
@@ -0,0 +1,42 @@
+#ifndef AMREX_EB_STL_UTILS_H_
+#define AMREX_EB_STL_UTILS_H_
```

Comment:
Could you add `#include <AMReX_Config.H>` after this line and do similar thing for the other new header too?  We just switched to a new style of including `AMReX_Config.H` in every header file, so that the macros defined during the generation of libamrex can be reliably used later.

Reply:
this is done